turbopack/crates/turbo-tasks-backend/src/kv_backing_storage.rs

```diff
@@ -552,21 +554,81 @@ fn process_task_data<'a, B: ConcurrentWriteBatch<'a> + Send + Sync>(
                     )
                     .entered();
 
+                    // The store the last task data and the last value as pointers to avoid looking
+                    // them up in the map again. Everytime we modify the map the pointers are
+                    // updated, so we never have a dangling pointer.
```

Comment:
```suggestion
                    // Store the last task data and the last value as pointers to avoid looking
                    // them up in the map again. Every time we modify the map the pointers are
                    // updated, so we never have a dangling pointer.
```